idea/src/org/jetbrains/kotlin/idea/quickfix/MoveConstValToCompanionObjectFix.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.quickfix
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import org.jetbrains.kotlin.diagnostics.Diagnostic
+import org.jetbrains.kotlin.idea.intentions.MoveMemberToCompanionObjectIntention
+import org.jetbrains.kotlin.psi.KtFile
+import org.jetbrains.kotlin.psi.KtProperty
+import org.jetbrains.kotlin.psi.psiUtil.getStrictParentOfType
+
+class MoveConstValToCompanionObjectFix(property: KtProperty) : KotlinQuickFixAction<KtProperty>(property) {
+
+    override fun getText() = "Move to companion object"
+
+    override fun getFamilyName() = text
+
+    override fun invoke(project: Project, editor: Editor?, file: KtFile) {
+        val property = element ?: return
+        MoveMemberToCompanionObjectIntention().applyTo(property, editor)
```

Comment:
Please look at the following line in QuickFixRegistrar:

```
PROPERTY_INITIALIZER_IN_INTERFACE.registerFactory(RemovePartsFromPropertyFix, ConvertPropertyInitializerToGetterIntention)
```

As I can see we are able to use intention as quick fix, and should not provide a wrapper. Try to do the same here